todo_app/tests/test_IntUnit.py

```diff
@@ -23,19 +17,33 @@ def client():
        with test_app.test_client() as client:
           yield client
 
-def get_requests():
-   mock_get_requests = [{"id": "5fb55f9084036928139db350", "title": "Testing", "status": "To Do", "dateLastActivity": "2020-11-18T18:43:33.434Z"}]
-   return mock_get_requests
+def test_viewModel():
+       item = [{'id':"testID", 'title':"testTitle1", 'status':"To Do", 'DateUpdated':"testDate"},
+       {'id': "testID", 'title': "testTitle2", 'status': "Doing", 'DateUpdated':"testDate"},
+       {'id': "testID", 'title': "testTitle3", 'status': "Done", 'DateUpdated':"testDate"}]
+       view_model = ViewModel(item)
+       view_model.statusToDo
+       view_model.statusDoing
+       view_model.show_all_done_items
+       assert len(view_model._ToDo) == 1
```

Comment:
These are now testing your app code, but it's a slightly odd use of the properties: we should avoid ever calling variables prefixed with `_` outside of the relevant class, that prefix suggests that outside users shouldn't need to know or care about that
We can replace that by just verifying the properties directly:
```
       assert len(view_model.statusToDo) == 1
       assert len(view_model.statusDoing) == 1
       assert len(view_model.show_all_done_items) == 1
```